misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java

```diff
@@ -141,7 +147,7 @@ static void execute(List<String> argList, boolean helpFlag, String moduleName, S
                                 project.currentPackage().module(FormatUtil.isModuleExist(project, moduleName));
                         try {
                             formattedFiles.addAll(iterateAndFormat(getDocumentPaths(project,
-                                    moduleToBeFormatted.moduleId()), sourceRootPath, dryRun));
+                                    moduleToBeFormatted.moduleId()), sourceRootPath, options, dryRun));
```

Comment:
codecov warning?

Reply:
check other warnings as well. May be we need to add some tests to cover these new lines.